### PR TITLE
Tinyxml2 recipe

### PR DESCRIPTION
**Pull request by gartung**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by gartung** [comment]
@chrisburr I don't know why the lint check won't clear even though the builds succeed.

File: recipes/tinyxml2/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  home: http://leethomason.github.io/tinyxml2/
+  license: zlib
+  license_file: LICENSE.txt
+  summary: 'TinyXML2 is a simple, small, efficient, C++ XML parser that can be easily integrated into other programs.'
+
+extra:
+  recipe-maintainers:
+    - gartung
+    - chrisburr
+    - henryiii
```
Comment (chrisburr) [inline recipes/tinyxml2/meta.yaml]: @henryiii Are you happy to be listed as a maintainer of this?

**Comment by chrisburr** [comment]
@gartung Are you happy with the changes I pushed?

File: recipes/tinyxml2/meta.yaml (hunk 1 above)
Comment (gartung) [reply recipes/tinyxml2/meta.yaml]: Yes

**Pull request merged**